### PR TITLE
Like button

### PR DESCRIPTION
**Pull request by AnnaGrasm**

### CODE REVIEW

**Review by JessicaLoers — COMMENTED** [review]
**Hello @AnnaGrasm** 👋
**Well done!**
I've some suggestions you could think about.

File: src/component/ButtonLike/ButtonLike.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export default function ButtonLike() {
+  const [isLiked, setIsLiked] = useState(false);
+
+  function toggleLike() {
+    setIsLiked(!isLiked);
+  }
+
+  return (
+    <StyledButton onClick={toggleLike} aria-label="like">
+      <StyledHeartIcon $isLiked={isLiked} />
```
Comment (JessicaLoers) [inline src/component/ButtonLike/ButtonLike.js:14]: Nice solution, but  take care of your props naming, please remove the `$` 😉
```suggestion
      <StyledHeartIcon isLiked={isLiked} />
```

**Review by JessicaLoers — COMMENTED** [review]
**Hi @AnnaGrasm** ✌️
there's  just one minor thing to change.

File: src/App.js [hunk 2]
```diff
@@ -38,4 +38,5 @@ const AppContainer = styled.div`
   gap: 1rem;
   min-height: 100vh;
   background-color: var(--primary-light-color);
+  position: relative;
```
[the remark this answers is not in the document]
Comment (AnnaGrasm) [reply src/App.js]: Thank you Jessica for the suggestions. I fixed all the point's you've mentioned  😁

File: src/component/ButtonLike/ButtonLike.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+const StyledButton = styled.button`
+  position: absolute;
+  right: 0.5rem;
+  top: 1rem;
+  background: transparent;
+  border: none;
+`;
+
+const StyledHeartIcon = styled(FaHeart)`
+  color: ${({$isLiked}) => ($isLiked ? '#662121' : '#e5cfcf')};
```
[the remark this answers is not in the document]
Comment (AnnaGrasm) [reply src/component/ButtonLike/ButtonLike.js:28]: Hey Jessica, I tried this out without $ but I always get this console message => `react-dom.development.js:86 Warning: React does not recognize the `isLiked` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isliked` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

I googled this issue and get this solution with $. What do you think?

[the remark this answers is not in the document]
Comment (JessicaLoers) [reply src/component/ButtonLike/ButtonLike.js:28]: Ah okay, I guess the problem lies in the combination of the attribute naming `isLiked` and the value, which is `true`
Take a look at my example in the comments before, where I used `active` as `truthy` not `true` or try to pass/use a string instead like this 👇 
````
  <StyledHeartIcon active={isLiked ? "true" : "false"} />
````

[the remark this answers is not in the document]
Comment (AnnaGrasm) [reply src/component/ButtonLike/ButtonLike.js]: unfortunately` <StyledHeartIcon active={isLiked ? "true" : "false"} />` didn't work. I tried `{isLiked ? <StyledHeartIcon active="true" /> : <StyledHeartIcon />}` and in styled Components this `color: ${({active}) => (active ? '#662121' : '#e5cfcf')};` and it works now 😊

File: src/component/ChairCard/ChairCard.js [hunk 4]
```diff
@@ -1,43 +1,46 @@
...
+    _id,
+    imgUrlAlternative1,
+    imgUrlAlternative2,
+    imgUrlAlternative3,
+  } = chair;
+
   return (
     <ChairItem role="chair-item">
       <ChairHeader>{name}</ChairHeader>
+      <ButtonLike />
```
Comment (macfrei) [inline src/component/ChairCard/ChairCard.js:25]: This button doesn't do anything as yet though 🤔

**Review by macfrei — COMMENTED** [review]
Good job, added my two cents ;)

File: src/component/ButtonLike/ButtonLike.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+export default function ButtonLike() {
+  const [isLiked, setIsLiked] = useState(false);
+
+  function toggleLike() {
+    setIsLiked(!isLiked);
+  }
+
+  return (
+    <StyledButton onClick={toggleLike}>
+      {isLiked ? <StyledHeartIcon active="true" /> : <StyledHeartIcon />}
```
[the remark this answers is not in the document]
Comment (AnnaGrasm) [reply src/component/ButtonLike/ButtonLike.js:14]: If I do this I get this in the console 

react_devtools_backend.js:4026 Warning: Received `true` for a non-boolean attribute `active`.

If you want to write it to the DOM, pass a string instead: active="true" or active={value.toString()}.

[the remark this answers is not in the document]
Comment (macfrei) [reply src/component/ButtonLike/ButtonLike.js:14]: Ah I see 🙈 my bad! Just ignore it^^

File: src/component/ChairCard/ChairCard.js (hunk 4 above)
Comment (macfrei) [reply src/component/ChairCard/ChairCard.js:25]: I missed that as well. Sorry Anna!

**Review by JessicaLoers — APPROVED** [review]
Well done Anna 🚀

**Review by JessicaLoers — APPROVED** [review]

**Pull request merged**